environment_set_up/QuickStart-HPC.md

```diff
@@ -0,0 +1,122 @@
+**3** ([running your notebooks](#3-run-notebooks)) for your routine workflow.
+
+This software environment refers to steps taken on the HPC host.
+
+### 1.a) Download Materials to HPC Account
+
+* Log in to the HPC machine
+* Run this command:
+
+```text
+curl -s -L 'https://github.com/hytest-org/hytest/releases/latest/download/HyTEST-Tutorials.zip' | tar xvf -
+```
+
+You should now have a new directory, `hytest` into which this archive has
+been placed.
+
+```text
+> cd hytest
+> ls -l
+...
+```
+
+### 1.b) Set Up a `hytest` Conda Environment
+
+> **NOTE**: We are using 'hytest' as the environment name.  If you need to use
+another name, you'll need to make some adjustments to the
+[environment file](./environment_set_up/HyTEST.yml).
+
+[Conda](https://docs.conda.io/en/latest/) is a software package manager -- it will automate much of the software downloading
+and configuring to satisfy software prerequisites to run HyTEST notebooks. Use the `HyTEST.yml` environment definition file
+as the input specification to create a new environment which includes the necessary software:
+
+```text
+> conda env create -f ./environment_set_up/HyTEST.yml
+...
+...
+```
+
+> **NOTE**: We're using the `conda` given to us from the HPC sysadmin. You may
+see other conda-like replacements used in similar environments elsewhere (`mamba` is a popular
+one).  They are functionally equivalent for what we need to do here.
+
+### 1.c) Configure Jupyter Server
+
+The Jupyter Server is the element which will actually run the notebooks on the HPC, but
+which display in a remote (i.e. your desktop) web browser.  We have an auto-configuration
+script which will help with this configuration. Run:
+
+```text
+> ./environment_set_up/auto-conf.py
+Set the access password for connecting to this Jupyter Server...
+Enter password:
+Verify password:
+...
+```
+
+The password you supply will be used when you attempt to connect to the
+Jupyter server from your local web browser.
+
+Note that this `auto-conf.py` script handles a lot of
+configuration that you need only do once. It writes a configuration
+file to `$HOME/.jupyter/jupyter_server_config.json`, which will simplify
+the process of starting the server in future steps.
+
+If you need (or want) to deviate from the vanilla set-up created by
+`auto-conf.py` you can find more detailed information for setting up
+manually [here](ManualConfig-HPC.md)
+
+With that, your jupter server and its necessary environment should be set.
+
+## 2) Start the Jupyter Server
+
+The computation and data access will run on the compute node, and render
+results via web connection to the browser on your desktop. This model
+uses a 'server' process on the HPC in order to coordinate that connection.
+
+Start Jupyter Server on an interactive compute node
+
+```text
+> salloc -A account_name -t 02:00:00 -N 1 srun --pty bash
+> conda activate hytest
+> jupyter lab
+```
+
+`account_name` is your account credential/name
```

Comment:
What account? is this my usgs email address?

Reply:
This is adapted from the text in here:
https://github.com/USGS-python/hytest-development/blob/main/hpc.md#installing-a-software-environment

it talks about needing to 'use your account name'... I presume this is the login name for getting access to the HPC server.